source/includes/webhooks/_scorecard_deleted.md

```diff
@@ -0,0 +1,30 @@
+## Scorecard deleted
+
+This web hook only fires when individual scorecards are destroyed.  This occurs when the Harvest API delete endpoint is used, or when the delete scorecard link is used in the application. This will not fire when a candidate or application is deleted. A candidate or application being deleted implies all their scorecards have been deleted with them.
```

Comment:
Do you think there's a historical reason why there isn't a `:dependent => :destroy` between application and scorecard?